assets/js/practice.js

```diff
@@ -12,6 +12,20 @@ const hardLvlBtn = document.getElementById("hardLvlBtn");
 
 const nameModalEle = document.querySelector(".nameModal");
 const startInstruction =  document.querySelector(".startIns")
+const links = document.querySelector(".navigation");
```

Comment:
add this code inside index.js` and not in other files.

Reply:
and link index.js in all pages.

Reply:
Updated